skrobot/utils/urdf.py

```diff
@@ -1254,6 +1254,9 @@ def __init__(self, name, origin, geometry):
         self.geometry = geometry
         self.name = name
         self.origin = origin
+        if _CONFIGURABLE_VALUES['force_visual_mesh_origin_to_zero']:
+            if self.geometry.mesh is not None:
```

Comment:
To correctly update the position of the collision mesh, it is better to apply the transformation as shown in the code below.
The `visualize-urdf` tool only displays the `visual` tag, but the collision mesh refers to the `collision` tag.
Therefore, the coordinates of the `collision` tag should also be transformed accordingly.
```
        if _CONFIGURABLE_VALUES['force_visual_mesh_origin_to_zero']:
            if self.geometry.mesh is not None:
                new_mesh = []
                for mesh in self.geometry.meshes:
                    mesh.apply_transform(self.origin)
                    new_mesh.append(mesh)
                self.geometry.mesh.meshes = new_mesh
                self.origin = np.eye(4)
                del new_mesh
```